src/olympia/blocklist/admin.py

```diff
@@ -534,6 +544,12 @@ def get_fieldsets(self, request, obj):
 
         return (details, history, edit)
 
+    def has_change_permission(self, request, obj=None):
+        if obj and BlockSubmission.get_submission_from_guid(obj.guid):
```

Comment:
Should there be a test for this, or is it considered covered by `test_cannot_edit_when_guid_in_blocksubmission`?

Reply:
that covers it - when `has_change_permission` is `False` the page is read only.